apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -523,6 +524,7 @@ var TopInstructions = React.createClass({
 module.exports = connect(function propsFromStore(state) {
   return {
     isEmbedView: state.pageConstants.isEmbedView,
+    embedViewLeftOffset: state.pageConstants.nonResponsiveVisualizationColumnWidth + 20,
```

Comment:
The existing margin on App Lab embed levels was a magic 20, so I kept it.  I'll extract a constant... somewhere.